sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/object.scala

```diff
@@ -286,6 +290,9 @@ case class FlatMapGroupsInR(
     child: LogicalPlan) extends UnaryNode with ObjectProducer{
 
   override lazy val schema = outputSchema
+
+  override protected def stringArgs: Iterator[Any] = Iterator(inputSchema, outputSchema,
+    outputObjAttr, child)
```

Comment:
I reverted it because I've noticed that I had a separate branch for that fix.
https://github.com/apache/spark/commit/939dbd5a17e63171ef2c18d5b23874daa75dbfcc

But if you think it is better to have it here, I can bring it back.
Thanks!